### ISSUE TITLE
Is HTTP Message Reporting enabled?

### ISSUE THREAD

**Issue by robbcrg** [duplicati/duplicati#2597]
If this is still experimental or not enabled just let me know. I am available to test this if it is available,

I have:
- [X ] searched open and closed issues for duplicates

----------------------------------------

### Version info
**Duplicati Version:**Same as the base install version: 2.0.1.61_experimental_2017-05-30
**Operating System:** Windows 10
**Backend:** Google Cloud Storage

### Bug description
I added the following options to try and trigger the HTTP reporting. At no point did I see any hits on my web-server during my Duplicati backups. The Web server responds to events from other systems including the Chrome browser on the same system as Duplicati using the same http URL.

After each phase I did a backup.

Phase 1:
send-http-any-operation : X
send-http-url: http://my.test.url

Phase 2; Added
send-http-level : All (default)

Phase 3; Added
send-http-message : (default)

Phase 4; running out of things to try
enable-module: sendHttpMessage
(since removed)

### Steps to reproduce
**Actual result:** No hits on web server, no unusual log entries (informational)
**Expected result:** The web server would be hit with something ...

**Comment by kenkendk** [comment]
The http module was contributed, so I have not tested myself, but I will investigate.

**Comment by robbcrg** [comment]
Thank you, I rather suspect I am doing something wrong.

**Comment by kenkendk** [comment]
I tested with the first version (I only set `--send-http-url`) and it worked.
Can you see if you get any errors if you go to `Show log` then `Live` and then set to `Profiling` ?

**Comment by dimkadv2** [comment]
Hello!

When I edit send-http-message, message has no change

in code i see next:

`            m_sendAll = Utility.Utility.ParseBoolOption(commandlineOptions, 
             if (string.IsNullOrEmpty(m_body))
                m_body = DEFAULT_MESSAGE;
`

`m_body` equalse `DEFAULT_MESSAGE` always.

so I'm added next code:

`commandlineOptions.TryGetValue(OPTION_MESSAGE, out m_body);` and it's works

Is it a bug or will there be some other implementation for this variable?

**Comment by robbcrg** [comment]
I reinstalled Duplicati (latest) and reviewed my settings. They URL had changed from what I had added to TRUE, I fixed that and removed another setting that seemed rogue. I then dumped the configs to review in my editor.  A simple save did NOT fix anything.

At this time my issue no longer exists. If anyone has an issue like this I recommend manually reviewing the configuration via a config export.

I apologize I did not capture the pre-config file as that might have been useful. I will add the config to the backup set.
